roles/device_roles/adsimdetector/examples/sim-cam-test/verify.yml

```diff
@@ -0,0 +1,40 @@
+---
```

Comment:
We should add a schema for these `verify` files to the `schemas` dir, and then add a CI check to make sure all our `verify` files are vaild.